src/App.tsx

```diff
@@ -16,79 +16,115 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
   ALPHABET,
   LENGTH,
 }
 
 type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
+  sort: SortType | null;
+  isReversed: boolean;
 };
 
-// Use this function in the render method to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+export const App: React.FC = () => {
+  const [updatedGoods, setUpdatedGoods] = useState<string[]>(goodsFromServer);
```

Comment:
In the task it was written 
```
// DON'T save goods to the state
// type State = {
//   isReversed: boolean,
//   sortType: SortType,
// };
```

This task is about making it with data from outside of component